iredis/client.py

```diff
@@ -241,9 +241,13 @@ def after_hook(self, command, command_name, args, completer):
             )
             if nativestr(select_result) != "OK":
                 raise ConnectionError("Invalid Database")
+            # When the connection is TimeoutError or ConnectionError, reconnect the connection will use it
+            self.connection.db = self.db
```

Comment:
I think it's should be `password` here
```suggestion
            self.connection.password = args[0]
```